Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```
@@ -1489,7 +1489,28 @@ script:
     - contextPath: Akamai.Enrollments.Change.Status
       description: Akamai enrollments change status.
       type: Dictionary
-  dockerimage: demisto/auth-utils:1.0.0.94075
+  - arguments:
+    - defaultValue: '0'
+      description: The change for this enrollment on which to perform the desired operation. Default is 0. "change_path" is used.
+      name: change_id
+      required: true
+    - defaultValue: '0'
+      description: Enrollment on which to perform the desired operation. Default is 0. "change_path" is used.
+      name: enrollment_id
+      required: true
+    - description: "Change path on which to perform the desired operation. Sample: /cps/v2/enrollments/100000/changes/88888888. Note: change_path is not listed in the reference as a parameter. However it can be extracted directly from \"list_enrollments_command\". This should be the most common useage when generate RestAPI's URL."
```

```suggestion
    - description: "Change path on which to perform the desired operation. Sample: /cps/v2/enrollments/100000/changes/88888888. Note: change_path is not listed in the reference as a parameter. However it can be extracted directly from \"list_enrollments_command\". This should be the most common usage when generating the RestAPI's URL."
```
Not sure if this is needed: 
Note: change_path is not listed in the reference as a parameter. 

Instead, i would remove the first part of the note, and only say:
The change path can be extracted directly from \"list_enrollments_command\". This should be the most common usage when generating the RestAPI's URL."